packages/main/src/components/protected/topbar/components/TopBarSearchModal.jsx

```diff
@@ -31,52 +38,73 @@ const TopBarSearchModal = () => {
     setKeys("");
     setValue("");
   };
+  const handleSearchType = type => {
+    setSearchType(type);
+  };
+
   const onSearchSubmit = async e => {
     if (e.keyCode === 13 && value.length >= 1) {
       setOpenSearch(true);
-      const getResult = async () => {
+      // const getResult = async () => {
+      //   try {
+      //     setLoading(true);
+      //     let response = await exactPlugin.apiCall(
+      //       user.org_id,
+      //       user._id,
+      //       value,
+      //       keys
+      //     );
+      //     if (response.status >= 200 || response.status <= 299) {
+      //       setResult(response.data.results.data);
+      //     }
+      //     setLoading(false);
+      //   } catch (e) {
+      //     setLoading(false);
+      //     console.error(e);
+      //   }
+      // };
+      // getResult();
+
```

Comment:
Remove this commented out code if its not being used

Reply:
It's not been used for now but will be used when the search endpoint is ready. 
I will remove it.